iosApp/iosApp/App/iOSApp.swift

```diff
@@ -0,0 +1,20 @@
+import SwiftUI
+
+@main
+struct iOSApp: App {
+	@StateObject var viewModel = iOSAppViewModel()
```

Comment:
These tabs look huge, no?

Reply:
Hmm not to me...I think you can set it in your github file viewer?  I use 4 width